gui/wxpython/gui_core/prompt.py

```diff
@@ -132,8 +132,9 @@ def _runCmd(self, cmdString):
         except UnicodeError:
             cmd = utils.split(EncodeString((cmdString)))
         cmd = list(map(DecodeString, cmd))
+        _cmd = {"cmd": cmd, "cmdString": str(cmdString)}
 
-        self.promptRunCmd.emit(cmd=cmd)
+        self.promptRunCmd.emit(cmd=_cmd)
```

Comment:
```suggestion
        self.promptRunCmd.emit(cmd=cmd_dict)
```

I'm not saying I like `cmd_dict`, but leading underscore is reserved for special cases, like protected/private attributes, so we need a proper name here.

Reply:
Yes I agree, fixed.